telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(Enum):
```

Comment:
@bomjacob Does this actually matter all that much? I know this might come off as rude, but I don't see why this matters. It's a really tiny thing.